src/ProjectTemplates/test/template-baselines.json

```diff
@@ -22,11 +22,11 @@
         "Pages/_ViewImports.cshtml",
         "Pages/_ViewStart.cshtml",
         "Pages/Shared/_Layout.cshtml",
+        "Pages/Shared/_Layout.cshtml.css",
         "Pages/Shared/_LoginPartial.cshtml",
         "Pages/Shared/_ValidationScriptsPartial.cshtml",
         "Properties/launchSettings.json",
         "wwwroot/favicon.ico",
-        "wwwroot/css/site.css",
```

Comment:
Do we want to keep a stub `site.css` file in the template or should we completely move over?

Reply:
I think we should keep it, it gives folks a place to put regular/global CSS.

Reply:
We generally don't ship empty files though. Particularly not one that isn't referenced by the app. Let's remove this, users can always add one back if they want to.